iOS/DittoChat.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -14,8 +14,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/getditto/DittoSwiftPackage",
       "state" : {
-        "revision" : "a7cc8dda460487e439e628b0a5044e86a983ad64",
-        "version" : "4.1.0"
+        "revision" : "63db2b032b5d3f6505ead0546e866a2caab401e2",
```

Comment:
We don't want to downgrade the iOS app from 4.1.0 to 4.0.0

Reply:
I'm not sure how/why Xcode project got touched. I did build from Xcode but nothing should have been pushed to iOS, this is a mistake on my part if it did